shared/actions/tracker.js

```diff
@@ -93,26 +93,45 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
   }
 }
 
-export function triggerIdentify (uid: string): TrackerActionCreator {
+export function getProfile (username: string): TrackerActionCreator {
+  return (dispatch, getState) => {
+    dispatch(triggerIdentify('', username, true, serverCallMap(dispatch, getState, true), false, true, true))
+  }
+}
+
+export function getMyProfile (): TrackerActionCreator {
+  return (dispatch, getState) => {
+    const status = getState().config.status
+    const myUID = status && status.user && status.user.uid
+    if (myUID) {
+      dispatch(triggerIdentify(myUID, '', true, serverCallMap(dispatch, getState, true), false, true, true))
+    }
+  }
+}
+
+export function triggerIdentify (uid: string = '', userAssertion: string = '', allowSelf: boolean = false, incomingCallMap: Object = {},
```

Comment:
this is reduced here: https://github.com/keybase/client/pull/3162/files#diff-418ae1d01939dbcc9c2ef8b3ab3d0e11R115